improver_tests/ensemble_copula_coupling/test_ResamplePercentiles.py

```diff
@@ -344,14 +341,14 @@ def test_check_single_threshold(self):
             [
                 [[4.0, 4.625, 5.25], [5.875, 6.5, 7.125], [7.75, 8.375, 9.0]],
                 [
-                    [24.44444444, 24.79166667, 25.13888889],
-                    [25.48611111, 25.83333333, 26.18055556],
-                    [26.52777778, 26.875, 27.22222222],
+                    [28.88889, 29.23611, 29.583334],
```

Comment:
What has changed is that at the point of calculating the percentiles, the source data had a 100th percentile of 50 Celsius (set in the test class) and this is now 60 Celsius (set by the ECC bounds) (lower bound moved from -50 to -100). This change was essential because the bounds are no longer passed between methods in the same way. As the result arises from linear interpolation from source data to upper bound, (or lower bound), then the results become more extreme.